packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -92,6 +104,9 @@ export const PaymentForm = ({
     [validator, onPayment, onPaymentError, stripe]
   );
 
+  /* istanbul ignore next */
```

Comment:
This should get covered in testing too, if only to assert that it's called with this string. Might also be good to move this string into a constant at the top of the file.